### PR TITLE
Added new profile components

### PR DESCRIPTION
**Pull request by dennisanthonyblabla**
Design changes according to the role of the user.

### CODE REVIEW

File: MacroChallengeTeam2/View/ProfileTanding.swift [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+//  Macro Challenge Team2
+//
+//  Created by Dennis Anthony on 11/10/22.
+//
+
+import Foundation
+import SnapKit
+import UIKit
+
+final class ProfileTanding: UIView {
```
Comment (m-salman-a) [inline MacroChallengeTeam2/View/ProfileTanding.swift]: Buat semua custom component atau view namanya belakangnya kasih "View" aja biar gampang dicari jadi ProfileTandingView

**Review by m-salman-a — APPROVED** [review]
Looks good 👍, just one small fix (see my comment)

File: MacroChallengeTeam2/View/ProfileTanding.swift (hunk 1 above)
Comment (m-salman-a) [reply MacroChallengeTeam2/View/ProfileTanding.swift]: Sama jangan lupa git convention buat commitnya, contoh:

Added UIColor Extension and also UIFont Extension.

jadi

feat: add extensions for UIColor and UIFont

Comment (dennisanthonyblabla) [reply MacroChallengeTeam2/View/ProfileTanding.swift]: oh bener juga, okee Man noted. hari ini gua kerjain

**Pull request merged**